src/databricks/labs/ucx/source_code/linters/context.py

```diff
@@ -21,7 +21,13 @@
 
 
 class LinterContext:
-    def __init__(self, index: MigrationIndex | None = None, session_state: CurrentSessionState | None = None):
+    def __init__(
+        self,
+        index: MigrationIndex | None = None,
+        session_state: CurrentSessionState | None = None,
+        dbr_version: tuple[int, int] | None = None,
+        is_serverless: bool = False,
```

Comment:
can we move `is_serverless` to `CurrentSessionState`? also, please modify the jobs.py to lint serverless jobs properly